src/signal_wrap.cc

```diff
@@ -112,21 +122,50 @@ class SignalWrap : public HandleWrap {
           wrap->MakeCallback(env->onsignal_string(), 1, &arg);
         },
         signum);
+
+    if (err == 0) {
+      CHECK(!wrap->active_);
+      wrap->active_ = true;
+      Mutex::ScopedLock lock(handled_signals_mutex);
+      handled_signals[signum]++;
+    }
+
     args.GetReturnValue().Set(err);
   }
 
   static void Stop(const FunctionCallbackInfo<Value>& args) {
     SignalWrap* wrap;
     ASSIGN_OR_RETURN_UNWRAP(&wrap, args.Holder());
+
+    if (wrap->active_)  {
+      wrap->active_ = false;
+      fprintf(stderr, "Stop %d\n", wrap->handle_.signum);
```

Comment:
I guess this (debug line) is a leftover?

Reply:
Yeah, thanks for noticing :) Removed it!